pdf_to_csv.py

```diff
@@ -156,13 +180,47 @@ def generate_page_table_data(self, input_pdf_filepath, input_pdf_obj,
             logger.warning(warning_message.format(page_num, input_pdf_filepath))
         return page_table_data
 
+    def log_data(self, input_pdf_filepath, page_num, measure_file_obj,
+                 column_coordinates):
+        '''Log additional data into a file.
+
+           Logs filename, page number, table coordinates, Whether table was
+           detected or not and column count
+
+           Args:
+               - input_pdf_filepath (string): The path of the pdf to be parsed.
+               - page_num (int): Page number that is being processed.
+               - measure_file_obj (obj: `file`): the file to write the
+                    information to, no data will be written incase this is
+                    None.
+               - column_coordinates (obj: `list of floats`): A list of
+                   coordinates where columns were detected.
+        '''
+        if measure_file_obj is None:
+            return
+        # The file is going to be a csv thus we are going to generate a string
+        # with the following information separated by `,`
+        # filename, page_num, table detected, column_coordinates,column count
+        # and write it to the file and add a newline.
+        filename = os.path.split(input_pdf_filepath)[-1]
+        table_detected = len(column_coordinates) > 1
+        parsed_column_coordinates = '|'.join([str(num) for num in
```

Comment:
Cool then let's define it as constant in the beginning MONITORING_LOG_SEPARATOR = "|" or PERF_LOG_SEPARATOR = "|" and use it as defacto for all the monitoring/performance logs?